topologies/binding/options.go

```diff
@@ -87,11 +87,11 @@ func (d *dialer) loadCertificates() (*x509.CertPool, tls.Certificate, error) {
 	}
 	trusBundle := x509.NewCertPool()
 	if !trusBundle.AppendCertsFromPEM(caCertBytes) {
-		return nil, tls.Certificate{}, fmt.Errorf("error in loading ca trust bundle")
+		return nil, tls.Certificate{}, err
```

Comment:
err here will be nil. was the intention to return no error here?

Reply:
addressed